src/yb/tserver/tablet_server.cc

```diff
@@ -1235,35 +1235,19 @@ std::vector<yb::util::WaitStateInfoPtr> TabletServer::GetThreadpoolWaitStates()
   return tablet_manager_->GetThreadpoolWaitStates();
 }
 
-std::vector<WaitStateInfoPB> TabletServer::ActiveUniverseHistory() const {
-  rpc::DumpRunningRpcsRequestPB dump_req;
-  rpc::DumpRunningRpcsResponsePB dump_resp;
-
-  dump_req.set_include_traces(false);
-  dump_req.set_get_wait_state(true);
-  dump_req.set_dump_timed_out(false);
-
-  WARN_NOT_OK(messenger()->DumpRunningRpcs(dump_req, &dump_resp), "DumpRunningRpcs failed");
-
-  std::vector<WaitStateInfoPB> res;
-  
-  for (auto conns : dump_resp.inbound_connections()) {
-    for (auto call : conns.calls_in_flight()) {
-      if (call.has_header() && call.header().remote_method().method_name() != "ActiveUniverseHistory") {
-        res.push_back(call.wait_state());
-      }
-    }
+void TabletServer::GetCQLServerMessenger(CQLServerMessenger messenger) {
```

Comment:
Should this be called Set instead of Get?

Also why is Messenger being sent by value? We should be storing a ptr, or a reference.

Reply:
I followed the example of `RegisterCertificateReloader` which seemed to do it this way.

Updating the Get to Set